gateway/src/network/network.routes.ts

```diff
@@ -68,4 +71,19 @@ export namespace NetworkRoutes {
       }
     )
   );
+
+  router.get(
+    '/tokens',
+    async (
+      req: Request<{}, {}, {}, TokensRequest>,
+      res: Response<TokensResponse, {}>
+    ) => {
+      try {
+        validateTokensRequest(req.query);
+        res.status(200).json(await getTokens(req.query));
+      } catch (error: any) {
+        res.status(error.status).json(error);
+      }
+    }
+  );
```

Comment:
I noticed that for some routes, we use `asyncHandler`(for I assume would be error handling), might I know why we are not using that for all routes?

Reply:
This I think we need to discuss during standup. But afaik, the try-catch does the equivalent as asyncHandler for route with async/awaitable calls.